### ISSUE TITLE
Is there a script to test the model with one image?

### ISSUE THREAD

**Issue by zhLawliet** [weinman/cnn_lstm_ctc_ocr#12]
now, I have trained a model with  dataset, how can I test the model with one image, which will output the result?

**Comment by anushaGundapaneni** [comment]
@zhLawliet  Do you have a script to test the model with one image ?

**Comment by zhLawliet** [comment]
no,i just output the all results one by one.

**Comment by anushaGundapaneni** [comment]
results one by one means ?

i just want to load a **text image** as input and that model should print the text present in the image.

have you checked with one image and is it printing the output ?

**Comment by zhLawliet** [comment]
I am sorry,i also want to get  the script.

**Comment by weinman** [comment]
@zhLawliet and @anushaGundapaneni, I also found such a script to be useful, so I pieced one together (much model loading is repeated from test.py, but the core loop and ops are sufficiently different). It's `validate.py` in the latest push (https://github.com/weinman/cnn_lstm_ctc_ocr/commit/cf64a7fba57479f001865193cc980577dc7e6d1c).

Let me know if you find this to work ok, and I will close this issue.
